dali/base/dautils.cpp

```diff
@@ -2038,6 +2039,15 @@ IRemoteConnection *getElementsPaged( IElementsPager *elementsPager,
             results.append(item);
         }
     }
+    if (total) {
+        try {
+            *total = elementsPager->getTotalElements();
+        }
+        catch(IException* e) {
+            EXCLOG(e, "elementsPager->getTotalElements()");
```

Comment:
I think you should assume implemented and not wrap.
Implementing some when is easy to implement all is complicating matters.